opal/tests/test_core_menus.py

```diff
@@ -107,15 +110,42 @@ def test_sets_items_from_app(self, get_app, plugin_list):
         menu = menus.Menu(user=self.user)
         self.assertEqual(menu_items, menu.items)
 
+    def test_excludes_items_from_app_if_for_user_is_false(
```

Comment:
Now quite bikeshedding, but I'd like these tests to explicitly call out that what they're really testing is `__init__`

Reply:
I'm not sure I understand, I don't want to call __init__ because I want is to make sure that this happens on initialisation.